### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

**Comment by kenvandine** [comment]
Actually this shows the old gitlab url

**Comment by jbicha** [comment]
@sergiusens Do you want to update this pull request based on Ken's comment?

**Comment by sergiusens** [comment]
Sure, I thought he went ahead and fixed it himself :sweat_smile:

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-git clone -b gnome-3-34-1804-sdk https://gitlab.gnome.org/Community/Ubuntu/gnome-sdk.git gnome-3-34-1804-sdk
\ No newline at end of file
+git clone -b gnome-42-2204-sdk https://gitlab.gnome.org/Community/Ubuntu/gnome-sdk.git gnome-42-2204-sdk
```
Comment (sergio-costas) [inline README.md]:
```suggestion
git clone -b gnome-42-2204-sdk https://github.com/ubuntu/gnome-sdk.git gnome-42-2204-sdk

```

**Review by sergio-costas — CHANGES_REQUESTED** [review]

**Pull request merged**